EasyPost/BatchCollection.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+
+namespace EasyPost
+{
+    public class BatchCollection
+    {
+        public List<Batch> batches { get; set; }
+        public bool has_more { get; set; }
```

Comment:
why is this in the snake case?

Reply:
All our JSON values are snake cased due to our deserialization